code-challenges.js

```diff
@@ -59,3 +154,12 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+function numbers (digit1, digit2) {
+    if(digit1 < digit2) { 
+        return digit2 - digit1 
+    } else // this is evaluating if digit1 is > digit 2
+    return digit1 - digit2   
+}
+console.log(numbers(number1, number2))
+console.log(numbers(number3, number4))
```

Comment:
Venessa!  This is great work!  Your pseudocode is fabulous.  Your code and syntax is great.  You checked your errors, provided your explanation, and provided corrected code!